backend/docker-compose.yml

```diff
@@ -0,0 +1,74 @@
+# For more information: https://laravel.com/docs/sail
```

Comment:
I was under the impression that Lando would've eliminated the need for this file.

Reply:
That docker compose comes with the Laravel project. I'm leaving it there for the time being as a reference until we decide how our production containers will be defined.